picturae_importer.py

```diff
@@ -920,8 +920,8 @@ def run_all_methods(self):
         # creating file list after conditions
         self.create_file_list()
 
-        # prompt
-        cont_prompter()
+        # prompt(uncomment for local or monitored imports)
```

Comment:
cont_prompter() commented out, not useful on a backend environment , only useful for a sandbox environment where a human monitor will want to stop to inspect the output before upload.